Source/EasyNetQ.ApprovalTests/EasyNetQ.approved.txt

```diff
@@ -460,6 +461,7 @@ namespace EasyNetQ
     public interface ISimpleConsumeConfiguration
     {
         EasyNetQ.ISimpleConsumeConfiguration WithArgument(string name, object value);
+        EasyNetQ.ISimpleConsumeConfiguration WithAutoAck();
```

Comment:
I remind that every such change is now breaking for the clients that implement their own custom `IXXXConfiguration`.